terraform/bootstrap/variables.tf

```diff
@@ -0,0 +1,2 @@
+variable "cf_password" {}
```

Comment:
[suggestion] I put {} for the variables in the overriding environments because the variables were already described in the shared module, but I wonder if it would make sense to add the description and type in this situation.  It's just a thought I had, but I'm not too worried about it particularly.